<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Level1</class>
 <widget class="QMainWindow" name="Level1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>404</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>190</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>190</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>190</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>190</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Level 1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../images/resource.qrc">
    <normaloff>:/img/cookie1.png</normaloff>:/img/cookie1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>386</x>
      <y>210</y>
      <width>151</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QPushButton" name="downArrow">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../images/resource.qrc">
         <normaloff>:/img/downarrow.png</normaloff>:/img/downarrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="upArrow">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../images/resource.qrc">
         <normaloff>:/uparrow.png</normaloff>:/uparrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="leftArrow">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../images/resource.qrc">
         <normaloff>:/leftarrow.png</normaloff>:/leftarrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="rightArrow">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../images/resource.qrc">
         <normaloff>:/rightarrow.png</normaloff>:/rightarrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QPushButton" name="jumpButton">
       <property name="text">
        <string>JUMP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="backToChoose">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбор уровня</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restartButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать заново</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="cookiesNumber">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="stepsNumber">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="steps">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ХОДЫ</string>
    </property>
   </widget>
   <widget class="QLabel" name="cookies">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ПЕЧЕНЬКИ</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridField"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../images/resource.qrc"/>
 </resources>
 <connections/>
</ui>
